insights/parsers/satellite_postgresql_query.py

```diff
@@ -0,0 +1,177 @@
+
+    Sample Output::
+
+                   name            |  url          |  value
+        ---------------------------+---------------+-----
+         abc                       |               | --- false+
+                                   |               | ...
+         def                       | http://xx.com | --- true +
+                                   |               | ...
+         (2 rows)
+
+    Examples::
+
+        >>> type(query)
+        <class 'insights.parsers.satellite_postgresql_query.SatellitePostgreSQLQuery'>
+        >>> 'abc' in query
+        True
+        >>> query.get_column_value('abc', 'value')
+        ['--- false+', '...']
+        >>> query.get_column_value('def', 'url')
+        'http://xx.com'
+
+    Raises::
+
+        SkipException: when there is no valid data in the table
+        ParseException: when the row is not matched to columns or
+                        the count of rows isn't the same with the
+                        row count output
+    """
+
+    def parse_content(self, content):
+        if not content:
+            raise SkipException("There is no data in the table")
+        self._columns = [item.strip() for item in content[0].split('|')]
+        self._first_columm = self._columns[0]
+        for row in content[1:]:  # skip the first column line
+            if row.strip(' +-'):
+                if row.strip().startswith('(') and row.strip().endswith('rows)'):
+                    try:
+                        self.count = int(row.split()[0].strip('('))
+                    except Exception:
+                        raise ParseException('The row count is not in int format')
+                    break
+                row_data = [item.strip() for item in row.split('|')]
+                if len(row_data) != len(self._columns):
+                    raise ParseException("Invalid line: '{0}'".format(row_data))
+                if row_data[0]:
+                    data = {}
+                    data[self._columns[0]] = row_data[0]
+                    data.update(dict(zip(self._columns[1:], [[] for i in range(1, len(self._columns))])))
+                    self.append(data)
+                    for i in range(1, len(self._columns)):
+                        if row_data[i]:
+                            data[self._columns[i]].append(row_data[i])
+                else:
+                    for i in range(1, len(self._columns)):
+                        if row_data[i]:
+                            data[self._columns[i]].append(row_data[i])
+        if not len(self):
+            raise SkipException("There is no data")
+        if self.count != len(self):
+            raise ParseException('The count of rows is not equal to the number of the last row')
+
+    def __contains__(self, first_column_value):
+        return True if any(
+            row[self._first_columm] == first_column_value for row in self
+        ) else False
+
+    def get_column_value(self, first_column_value, column_name):
```

Comment:
@bfahr, to make it more readable, I redefine it with "keyword_search". In this way, the user can define the search conditions by himself to find one row. Since the most usage is to get the column of one row. So It is limited to one row only.